op-e2e/actions/helpers/env.go

```diff
@@ -50,9 +50,9 @@ func WithActiveGenesisFork(fork rollup.ForkName) EnvOpt {
 }
 
 // DefaultFork specifies the default fork to use when setting up the action test environment.
-// Currently manually set to Holocene.
+// Currently manually set to Jovian.
 // Replace with `var DefaultFork = func() rollup.ForkName { return rollup.AllForks[len(rollup.AllForks)-1] }()` after Interop launch.
-const DefaultFork = rollup.Holocene
+const DefaultFork = rollup.Jovian
```

Comment:
If you prefer, there's ways to set just the jovian tests to use jovian. I do that here:

https://github.com/ethereum-optimism/optimism/pull/17034/files

I think this is fine though.